config.json

```diff
@@ -219,7 +219,13 @@
       "core": false,
       "difficulty": 4,
       "slug": "diamond",
-      "topics": null,
+      "topics": [
+		"strings",
+		"loops",
+		"arrays",
+		"lists",
+		"text_formatting"
+		] ,
```

Comment:
Great suggested topics! :) Just a very minor thing, there shouldn't be a space here after the square bracket and before the comma. Would you mind removing it? :)